core/templates/dev/head/pages/exploration-editor-page/improvements-tab/improvements-tab.directive.html

```diff
@@ -56,10 +55,36 @@ <h2 class="oppia-improvement-card-header-title protractor-test-improvements-card
       <button ng-repeat="actionButton in card.getActionButtons()"
               ng-click="actionButton.execute()" ng-disabled="!actionButton.isEnabled()"
               ng-class="actionButton.getCssClass()"
-              class="btn protractor-test-improvements-action-button"
-              style="margin-left: 15px" type="button">
+              class="btn test-improvements-action-button"
+              type="button">
         <[actionButton.getText()]>
       </button>
     </div>
   </md-card>
 </div>
+<style>
+  .aggregated-stats-supplemental {
+    padding-top: 30px;
+    margin-bottom: 30px;
+  }
+  .stats-card-supplemental {
+    text-align: center;
+    padding: 0;
+  }
+  .option-card-supplemental {
+    padding: 10px; margin: auto
+  }
+
+  .option-card-text {
+    font-size: 14px;
+  }
+  .test-improvements-action-button {
+  margin-left: 15px;
+  }
+  improvements-tab-dashboard-supplemental {
+    position: relative;
+  }
+  .test-improvements-open-cards-count {
+    margin: 0;
+  }
+</style>
```

Comment:
Add a newline below

Reply:
I have added a new line below.

Reply:
@hungphi98, I think @kevinlee12 asked you to add a new line after `</style>`.